config/config.go

```diff
@@ -136,6 +140,15 @@ var (
 	PelicanVersion string
 )
 
+func (e *MetaDataErr) Error() string {
+	return e.Err
+}
+
+func (e *MetaDataErr) Is(target error) bool {
```

Comment:
This makes the error a single-use: all metadata errors are considered the "same".

Have `MetadataErr` wrap the underlying error.  This allows things like stack traces to propagate through and logic on the original error cause.  That is, add an `Unwrap` error that returns the original cause (see https://pkg.go.dev/errors).